src/include/esp-idf/bindings.h

```diff
@@ -15,6 +15,20 @@
 #include "esp_ipc.h"
 #include "esp_mac.h"
 
+#include "freertos/task_snapshot.h"
+
+#if CONFIG_IDF_TARGET_ESP32
+#include "esp_himem.h"
```

Comment:
`esp_himem.h` seems not available on ESP IDF 5.0. Even on 4.4, using `esp_himem.h` outputs a warning, that you should instead include `esp32/himem.h`. So:
* Please replace `esp_himem.h` with `esp32/himem.h`
* Since on 5.0 `himem.h` is part of the psram component, please protect this header just like you did protect the psram header.